src/js/modules/infragistics.ui.editors.js

```diff
@@ -9198,6 +9199,14 @@
 				}
 			}
 		},
+		_parseValueByIMEMask: function() {

```

Comment:
make the function with parameter - so at least this could be unit tested. Because the

Reply:
Also because hard-coded `this._currentCompositionValue` is not very reusable.